### PR TITLE
Agreement History improvements

### PR DESCRIPTION
**Pull request by stevtek**
improvements to Agreements History from feedback after demo

## What changed

* change log titles and messages to better match design (using a summary instead of the user in the title, etc)
* add scrollbars to MacOS

## Issue

#1300 

## How to test

edit an agreement and it's BLIs and view the history

## Screenshots

image

### CODE REVIEW

**Review by fpigeonjr — APPROVED** [review]

File: frontend/src/components/Agreements/AgreementDetails/AgreementHistoryList.js [hunk 1]
```diff
@@ -11,17 +11,32 @@ const findObjectTitle = (historyItem) => {
     }
 };
 
+const logItemTitle = (historyItem) => {
+    const className = convertCodeForDisplay("className", historyItem.class_name);
+    const objectTitle = findObjectTitle(historyItem);
```
Comment (tdonaworth) [inline frontend/src/components/Agreements/AgreementDetails/AgreementHistoryList.js:16]: Can we resolve the linting issues?

**Review by tdonaworth — APPROVED** [review]

File: frontend/sass/uswds/styles.scss [hunk 2]
```diff
@@ -123,3 +123,37 @@ $themeColors: (
...
     justify-content: space-between;
 }
+
+// Force scrollbars on MacOS with this class, force-show-scrollbars
+// based on https://gist.github.com/pjobson/c45f9c3dfa4b7fc3f30f0de29d1beaa1
+// but without the JS for hover/transition effects
+// There is simpler global option here: https://simurai.com/blog/2011/07/26/webkit-scrollbar
+.force-show-scrollbars::-webkit-scrollbar-track:vertical {
+    border-left: 1px solid #E7E7E7;
+    box-shadow: 1px 0 1px 0 #F6F6F6 inset, -1px 0 1px 0 #F6F6F6 inset;
```
Comment (fpigeonjr) [inline frontend/sass/uswds/styles.scss:133]: wonder if we should add these css hex codes as css variables

File: frontend/src/components/Agreements/AgreementDetails/AgreementHistoryList.js [hunk 3]
```diff
@@ -11,17 +11,30 @@ const findObjectTitle = (historyItem) => {
...
+        return `${className} Updated`;
+    } else if (historyItem.event_type === "DELETED") {
+        return `${className} Deleted`;
+    }
+    return `${className} ${historyItem.event_type}`;
+};
+
 const summaryMessage = (historyItem) => {
     const className = convertCodeForDisplay("className", historyItem.class_name);
+    const classNameLower = className.toLowerCase();
```
Comment (fpigeonjr) [inline frontend/src/components/Agreements/AgreementDetails/AgreementHistoryList.js:28]: this is a candidate for to be added as a utility function in our helpers dir

File: frontend/src/components/Agreements/AgreementDetails/AgreementHistoryList.js [hunk 4]
```diff
@@ -11,17 +11,30 @@ const findObjectTitle = (historyItem) => {
...
+    const classNameSentence = classNameLower.charAt(0).toUpperCase() + classNameLower.slice(1);
     const userFullName = historyItem.created_by_user_full_name;
```
Comment (fpigeonjr) [inline frontend/src/components/Agreements/AgreementDetails/AgreementHistoryList.js:30]: also we have a custom hook `useGetUserFullNameFromId.js`  but may not be applicable here

**Review by fpigeonjr — APPROVED** [review]
spun up locally and the scrollbars are overriding my os preferences
- added some non-blocking suggestions

File: frontend/src/components/Agreements/AgreementDetails/AgreementHistoryList.js (hunk 1 above)
Comment (stevtek) [reply frontend/src/components/Agreements/AgreementDetails/AgreementHistoryList.js]: done

**Pull request merged**